cloud-platform/app-deploy.md

```diff
@@ -0,0 +1,205 @@
+---
+category: cloud-platform
+expires: 2018-01-31
+---
+# Deploying an application to the Cloud Platform
+
+## Overview
+
+The aim of this guide is to walkthrough the process of deploying an application into the Cloud Platform.
+
+This guide uses a pre-configured application as an example of how to deploy your own.
+
+## Prerequisites
+
+This guide assumes the following:
+
+* Kubectl is installed and configured.
+* Docker is installed and configured.
+* Authentication with the cluster has been established.
+* An environment has been created within the Cloud Platform.
+* Access to AWS, with ECR upload permissions.
+* AWS CLI configured with account credentials.
+
+If you are not deploying your own application and would like to deploy the example application, clone the following repo:
+
+[https://github.com/ministryofjustice/cloud-platform-demo-app](https://github.com/ministryofjustice/cloud-platform-demo-app)
+
+## Pushing application to ECR
+
+To deploy an application to the Cloud Platform, firstly the application image needs to be retrievable from a repository.
+
+Amazon's ECR, within ECS is where all of the application images used by the Cloud Platform are stored.
+
+### Creating or choosing a repository
+
+On the ECR start page you will have the option create a new repository, or to search for an existing one.
+
+Create a new repository, and name it something relevant:
```

Comment:
Should we instead/also provide instructions on how to do this on the CLI, like the steps below?

Reply:
Can do. I will have to learn how to do this myself first.